### ISSUE TITLE
[Events] past event showing as Featured

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#10200]
**BLOCKED on #10176?**

## Describe the defect
This _might_ be resolved. We cannot prove it without locally standing up mock data to verify, with various events/moments in time.

- [Upcoming events](https://www.va.gov/minneapolis-health-care/events/): shows next chrono event on 8/12 2p
- [Past events](https://www.va.gov/minneapolis-health-care/events/?selectedOption=past): shows last chrono on 8/9 2p
- [Date filter for 8/11](https://www.va.gov/minneapolis-health-care/events/?selectedOption=specific-date&startDateDay=11&startDateMonth=08&startDateYear=2022): shows no events
but:
- [MN featured/fallback events ](https://www.va.gov/minneapolis-health-care/)shows an event on 8/11 10a
- Event node: https://prod.cms.va.gov/node/27142/edit

## AC / Expected behavior
- [ ] Create mock data locally to prove that when no event is featured, the next chrono event / recurrence appears
- [ ] Create an automated test to protect this functionality. (Need to evaluate how content-build factors in, and how staleness affects this.)
- [ ] Past Events shouldn't show as VAMC featured when no event is featured. `Content-build` fix. It _could_ be that the event showing here is actually featured. But Featured events in the past should still not show.
- [ ] Past Event should should show under Past Events and be findable via Specific Date filter. `vets-website` fix. If we need to break this into a separate ticket, we can.

## Screenshots
![Screen Shot 2022-08-11 at 2 23 12 PM](https://user-images.githubusercontent.com/85581471/184244912-0dc0389c-227b-413e-b53f-32514fad2df7.png)
![Screen Shot 2022-08-11 at 2 23 34 PM](https://user-images.githubusercontent.com/85581471/184244919-866fff94-0ead-4a2c-9f50-9925de92b62b.png)
![Screen Shot 2022-08-11 at 2 23 41 PM](https://user-images.githubusercontent.com/85581471/184244921-1e0e743f-39d7-4f09-a11c-535d607a6acf.png)
![Screen Shot 2022-08-11 at 2 24 00 PM](https://user-images.githubusercontent.com/85581471/184244922-0afb4aa1-0ed9-4815-9b9a-e44afa421892.png)

 ## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by jilladams** [comment]
First step here is to just triple confirm that code is deployed for https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9576. I'm still pretty unclear on whether that's live and this repros or isn't actually live yet. Recordig since I'm on PTO from nowish.

Slack thread: https://dsva.slack.com/archives/C52CL1PKQ/p1660249476638739

**Comment by jilladams** [comment]
As of this morning, the next chrono event from Upcoming events also shows as the Featured event on MN page. I'll keep an eye on this for awhile just to confirm no weirdness before closing: 
![Screen Shot 2022-08-17 at 8 48 44 AM](https://user-images.githubusercontent.com/85581471/185184728-4d668c58-64a4-4011-8481-63a2e8f0cf4c.png)
![Screen Shot 2022-08-17 at 8 48 50 AM](https://user-images.githubusercontent.com/85581471/185184733-44733f5f-b9be-4cb5-a9bf-ac61a10d4c5e.png)

**Comment by FranECross** [comment]
@jilladams It looks like from your last comment that this might be fixed? :)

**Comment by jilladams** [comment]
@FranECross Ugh, unfortunately I didn't confirm at the time and it's been too long now to trust that guess last year. It would probably be best to have engineering do the legwork with mock data to make sure-sure.

**Comment by FranECross** [comment]
chris to verify this as part of the ticket he's currently working on.

**Comment by FranECross** [comment]
Closing as no op; we have a ticket to be worked on soon where all past events will be hidden.

**Comment by jilladams** [comment]
Per refinement discussion: on VAMCs, Events show up as featured when they use the Featuring mechanism on the Event form, in the right sidebar: 
![Screenshot 2024-07-31 at 12 25 26 PM](https://github.com/user-attachments/assets/d2fcfda9-227b-402c-bb59-ace528a85d5b)

Featuring has its own problems, described in the epic [Deprecate "featured" UX and combined workflow for stories and events #1318](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/1318). We don't want to fix featuring on Events, we eventually want to remove it from Events entirely, and update the mechanism for how Events appear in the Featured space on VAMCs. 

But in the meantime: it may be possible for an event that was Featured to elapse, be in the past, and still show as featured. This ticket is currently closed as a no op. I don't think we want to reopen it, since we don't want to invest time on the featuring mechanism anyway. But: it's not fixed, and the work regarding past events filtering on listing pages will not affect this issue.

**Comment by FranECross** [comment]
@jilladams Thanks so much for this information. I've added a task for myself to grab this info and broach the subject to see if we want to deprecate 'featured' as part of the Events Future-State work.
